monkey/infection_monkey/ransomware/ransomware_payload.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+LOG = logging.getLogger(__name__)
```

Comment:
In some places we're using `logger = logging...`. In others we're using `LOG = logging...`. We should pick one and stick to it. I prefer `logging`, since I generally expect a constant when I see something in all caps. This is open for debate, though.